src/pages/SearchPage.js

```diff
@@ -62,11 +59,10 @@ class SearchPage extends Component {
     constructor(props) {
         super(props);
 
-        Timing.start(CONST.TIMING.SEARCH_RENDER);
```

Comment:
We still want the Timing I believe, can we get this added back but maybe put where it belongs now?